test/browser/6-mya.cy.test.ts

```diff
@@ -119,4 +121,33 @@ describe('Welsh Manage your appeal app @mya', () => {
     expect(result.issues.length).to.equal(0, JSON.stringify(result.issues, null, 2));
   });
 
+  describe('CY - Hearing page', () => {
+    it('CY - Navigate to hearing tab', async() => {
+      statusPage.verifyPage();
+      await statusPage.clickElement('#tab-hearing');
+      await page.waitFor(500);
+      expect(await statusPage.getElementText('.navigation-tabs ul li.selected')).contain(content.cy.hearingTab.tabHeader);
+    });
+  });
+
+  describe('CY - Audio/video Evidence page', () => {
+    it('CY - Navigate to Audio/Video Evidence tab', async() => {
+      await statusPage.clickElement('#tab-avEvidence');
+      await page.waitFor(500);
+
+      expect(await statusPage.getElementText('.navigation-tabs ul li.selected')).contain(content.cy.avEvidenceTab.tabHeader);
+      expect(await statusPage.getElementText('.task-list div div')).contain(content.cy.avEvidenceTab.noEvidence);
+    });
+  });
+
+  xdescribe('CY - Appeal Details page', () => {
```

Comment:
typo

Reply:
That test requires further investigation as it's continuously failing and there is a JIRA ticket created on TA board .

Reply:
Therefore the test has been excluded from regression.